Add interfaces/local-font-access.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/local-font-access.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389

stephenmcgruer commented:
From https://github.com/web-platform-tests/wpt/pull/24210:

> Implementation is in progress in Chrome, it's behind a flag and the API shape is in flux. I wouldn't bother to add IDL tests just yet.

So let's let reffy keep updating this open PR until the API stabilizes and Chrome ships it behind experimental web platform features, then we can land a test for it.

autofoolip committed:
Add interfaces/local-font-access.idl

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/local-font-access.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389
